frontend/src/container/SearchResult.jsx

```diff
@@ -142,27 +142,27 @@ export class SearchResult extends React.Component {
 
   handleClickSeeMore = async () => {
     const { props } = this
-    const NEXT_PAGE = props.searchResult.currentNumberPage + 1
+    const NEXT_PAGE = props.simpleSearch.currentNumberPage + 1
```

Comment:
Why the naming in uppercase here?

Reply:
I think it is to have the same syntax as FIRST_PAGE, but this is old code, I am not sure. @grignards did you find nextPage more coherent? I'm not against.

Reply:
In my head an uppercase const is a constant (is never changed during the execution of the program), here I would have named it in camel-case like you suggest.